portfolio/src/main/java/com/google/sps/servlets/FollowedTagsServlet.java

```diff
@@ -51,37 +51,32 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/follow-tags")
 public class FollowedTagsServlet extends HttpServlet {
-    List<String> followedPosts= new ArrayList<>();
+  List<String> followedPosts= new ArrayList<>();
```

Comment:
**nit:** leave space between followedPosts and =

Reply:
Also, please put explicit public/private/protected annotator for fields.